fab/utils.py

```diff
@@ -90,34 +91,31 @@ def tag_commit(self):
                 'date': datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'),
             }
         )
-        self._deploy_tag = tag_name
+        self._tag_created = True
 
     def _offline_tag_commit(self):
         commit = local('cd {}/commcare-hq && git show-ref --hash --heads {}'.format(
             OFFLINE_STAGING_DIR,
-            env.deploy_metadata.deploy_ref,
+            self.deploy_ref,
         ), capture=True)
 
-        tag_name = '{}-{}-offline-deploy'.format(self.timestamp, self._environment)
         local('cd {staging_dir}/commcare-hq && git tag -a -m "{message}" {tag} {commit}'.format(
             staging_dir=OFFLINE_STAGING_DIR,
             message='{} offline deploy at {}'.format(self._environment, self.timestamp),
-            tag=tag_name,
+            tag=self._deploy_tag,
             commit=commit,
         ))
 
         with settings(warn_only=True):
             local('cd {staging_dir}/commcare-hq && git push origin {tag}'.format(
                 staging_dir=OFFLINE_STAGING_DIR,
-                tag=tag_name,
+                tag=self._deploy_tag,
             ))
+        self._tag_created = True
 
     @property
     def diff_url(self):
-        if env.offline:
-            return '"No diff url for offline deploy"'
```

Comment:
This gets tricky when you remove this, i don't think `self._last_tag` exists for an offline deploy since it searches for the last tag on github. however this could just be a remnant of me not being able to access outside internet from my local machine while on the vpn... so i think probably fine to keep